run_nextflow_RNAseq.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+workflow_path='/hpc/diaggen/users/lonneke/github/DxNextflowRNA'
+
+# Set input and output dirs
+input=`realpath $1`
+output=`realpath $2`
+email=$3
+
+mkdir -p $output && cd $output
+mkdir -p log
+
+if ! { [ -f 'workflow.running' ] || [ -f 'workflow.done' ] || [ -f 'workflow.failed' ]; }; then
+touch workflow.running
+echo "check directory for output: ${output}"
+
+export JAVA_HOME='/hpc/diaggen/software/tools/jdk-18.0.2.1/'  # change java version 
+export NXF_JAVA_HOME='/hpc/diaggen/software/tools/jdk-18.0.2.1/'  # change java vesion of nextflow
```

Comment:
Would not work with `export=NONE`. In my opinion general exports hidden in scripts like this is dangerous and should be avoided. Can we have a look at this and fix?

Reply:
I will pick this up.